shell-completion/bash/meson.build

```diff
@@ -10,7 +10,7 @@ if bashcompletiondir == ''
         endif
 endif
 
-if bashcompletiondir != 'no'
+if bashcompletiondir != 'no' and populate_root
```

Comment:
isn't it better to change bashcompletiondir rather than adding an additional check? would be mor eintuitive